_topic_map.yml

```diff
@@ -521,6 +521,8 @@ Topics:
   - Name: Diagnostics Tool
     File: diagnostics_tool
     Distros: openshift-origin,openshift-enterprise
+  - Name: Idling Applications
+    File: idling_applications.adoc
```

Comment:
@bfallonf Just noticed the `File` has an `.adoc` on the end here. Looks like this is building in asciidoctor but not sure how it will fare with Portal.